### DISCUSSION TITLE
Is there any way to have the client connect via domain name instead of IP?

### DISCUSSION THREAD

**Discussion by MrMeapz** [jchristn/WatsonWebsocket#81] [Q&A]
Hi.

I am able to connect my client to the server using a public IP, however I would like to connect via domain name instead, for example:
![image](https://user-images.githubusercontent.com/32072093/150586713-7213480a-1d4a-4e75-bfc6-1d71c65fa2f3.png)
This does not work for me.

I am running a nginx webserver on Debian.

Thanks in advance,

**Comment by MrMeapz** [comment]
I am trying to use it with Cloudflare for a reverse proxy, hence the question.
I deal with daily ddos attacks.

**Comment by jchristn** [comment]
Hi @MeapVR I'm very sorry for the delay.  Have you tried amending the test project to use the ```Uri``` constructor variant?  That should do the trick.
